semih-week2.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+const isOdd = x => x % 2 !== 0;
+const double = x => x * 2;
+const doubleOdd = numbers.filter(isOdd).map(double);
```

Comment:
While this code works fine it is more common to put the different functions in different `const`. So instead of a const with `x => x % 2 !== 0` and using that const in the `filter()` you could also write it together like this

`const isOdd = numbers.filter(x => x % 2 !== 0);`

For the doubleOdd you can then use the isOdd value to do the `map()`. Again, not wrong but it is more readable to put the things together.